### DISCUSSION TITLE
Add last login field

### DISCUSSION THREAD

**Discussion by longlee218** [accounts-js/accounts#1214] [Q&A] [answered]
How to set or add field loginAt/lastLogin in accountjs

**Comment by pradel** [comment, marked as answer]
You can use the `.on` event https://www.accountsjs.com/docs/api/server/classes/accountsserver#on
In the event update the field on the user with a DB call.

**Reply by longlee218** [reply]
thanks,  it's really good

**Comment by longlee218** [comment]
I found my answer. If you want to add field/update field loginAt or whatever you call. You can see suggestions below.
![image](https://user-images.githubusercontent.com/54740478/156356955-188d9c26-3510-4aff-9eab-56faf1cae2d8.png)
